servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -163,10 +163,8 @@ def query_jobs(body):
     """
     query = QueryJobsRequest.from_dict(body)
 
-    page_size = query.page_size
-    if not page_size:
-        page_size = _DEFAULT_PAGE_SIZE
-    offset = page_tokens.decode(query.page_token)
+    page_size = query.page_size or _DEFAULT_PAGE_SIZE
```

Comment:
Nice change here, we would change the Cromwell controller to this one-liner as well.